hpxml_version_translator/converter.py

```diff
@@ -701,8 +701,8 @@ def get_pre_post_from_building_id(building_id):
                 )
         else:
             raise exc.HpxmlTranslationError(
-                f"{hpxml2_file.name} was not able to be translated "
-                f"because 'AtticType' of {this_attic.SystemIdentifier.attrib['id']} is unknown."
+                f"{pathobj_to_str(hpxml2_file)} was not able to be translated "
+                f"because 'AtticType' of {this_attic.SystemIdentifier.attrib['id']} is not provided."
```

Comment:
If `hpxml2_file` is a string, then `hpxml2_file.name` throws a python error. Now it correctly throws our error message.